examples/service-clients/odsp-client/shared-tree-demo/.eslintrc.cjs

```diff
@@ -5,5 +5,27 @@
 
 module.exports = {
 	extends: [require.resolve("@fluidframework/eslint-config-fluid/recommended"), "prettier"],
-	rules: {},
+	rules: {
+		/**
+		 * Allow Fluid Framework examples to import from unstable APIs.
+		 * https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-internal-modules.md
+		 */
+		"import/no-internal-modules": [
+			"error",
+			{
+				allow: [
+					// Within Fluid Framework allow import of '/beta' from other FF packages.
```

Comment:
Are these rules not captured by our shared config? Do we have work tracking updating these settings there?

Reply:
Our rules specify /internal is allowed. That is the preferred setting for product code. For examples, we want /beta, /alpha, and /legacy. There are no rules that allow that. Also, I don't know of a way to modify this rule instead of just overwriting it, unless we get more scripty with the config.